dev/com.ibm.ws.security.token.ltpa/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -48,5 +55,5 @@ fileName.desc=The name of the file that contains the LTPA validation keys. The p
 password=LTPA validation keys password
 password.desc=The password for the LTPA validation keys. The best practice is to encrypt the password by using the securityUtility tool.
 
-notUseAfterDate=Not use after date
-notUseAfterDate.desc=A date and time value in ISO date format, after which the LTPA validation keys are no longer used. The following example shows the ISO date format: "2023-04-18T18:08:35Z". If the value is not specified, the LTPA validation keys can be used forever.
\ No newline at end of file
+validUntilDate=LTPA validation keys valid until date
+validUntilDate.desc=A date and time value in ISO date format, that the LTPA validation key is valid until. After the specified time, the validation key will no longer be accepted for aunthentication. The following example shows the ISO date format: "2025-11-18T18:08:35Z". If the value is not specified, the LTPA validation keys can be used forever.
```

Comment:
Change
A date and time value in ISO date format, that the LTPA validation key is valid until. After the specified time, the validation key will no longer be accepted for aunthentication.
to
A date and time value in ISO date format that the LTPA validation key is valid until. After the specified time, the validation key is no longer accepted for authentication.